integration-tests/validator-tool_test.go

```diff
@@ -0,0 +1,94 @@
+package integrationtests
```

Comment:
How about `package integration`?

Can you rename the file path to `integration/validator_tool_test.go`? Let's keep with all underscores.